Example/Controllers/ContactsListSettingsTableViewController.swift

```diff
@@ -228,20 +228,20 @@ class ContactsListSettingsTableViewController: UITableViewController {
     }
 
     class func fakeMail(with name: String?) -> String {
-        name != nil ? name!.replacingOccurrences(of: " ", with: ".", options: .literal, range: nil) + "@example.com" : ""
+        name != nil ? name!.replacingOccurrences(of: " ", with: ".", options: .literal, range: nil).lowercased() + "@example.com" : "".lowercased()
```

Comment:
you don't have to lower case `""` ^^

```suggestion
        name != nil ? name!.replacingOccurrences(of: " ", with: ".", options: .literal, range: nil).lowercased() + "@example.com" : ""
```